ansible-scylla-node/tasks/common.yml

```diff
@@ -48,6 +48,18 @@
       when: cpuset_command is defined and skip_cpuset is defined and skip_cpuset|bool == false
   become: true
 
+- name: Detect NVME disks
+  block:
+    - name: Get list of NVMEs
+      shell: |
+        nvme list | awk '{print $1}' | tail -n +3
```

Comment:
`nvme` is not installed by default. You need to install it before you can use it.

Reply:
On Ubuntu this is an `nvme-cli` package for example.

Reply:
Other than that - very nice idea.

Reply:
I'm now installing `nvme-cli` for both, Debian and RH.